### PR TITLE
Add text restore after restore icon

### PR DESCRIPTION
**Pull request by GretaD**
Deleted files have text "Restore" after the restore icon. We should have it also for deleted shares

### CODE REVIEW

File: apps/files_sharing/js/app.js [hunk 1]
```diff
@@ -264,7 +264,7 @@ OCA.Sharing.App = {
 		const fileActions = new OCA.Files.FileActions()
 		fileActions.registerAction({
 			name: 'Restore',
-			displayName: '',
+			displayName: 'Restore',
```
Comment (rullzer) [inline apps/files_sharing/js/app.js]: shouldn't it be translated?

Comment (GretaD) [reply apps/files_sharing/js/app.js]: damn, yes

**Review by rullzer — APPROVED** [review]

**Review by juliusknorr — APPROVED** [review]

**Comment by juliushaertl** [comment]
Bundles also need an update :wink: 

```
Uncommited changes in webpack build
HEAD detached at pull/19435/merge
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   apps/files_sharing/js/dist/files_sharing.js
	modified:   apps/files_sharing/js/dist/files_sharing.js.map

no changes added to commit (use "git add" and/or "git commit -a")
```

**Comment by juliushaertl** [comment]
@GretaD Mind to backport this manually?

**Pull request merged**